io-engine/src/bdev/nexus/nexus_persistence.rs

```diff
@@ -1,7 +1,12 @@
 use super::{IoMode, Nexus, NexusChild};
-use crate::{persistent_store::PersistentStore, sleep::mayastor_sleep};
+use crate::{
+    persistent_store::{to_json_byte_vec, PersistentStore},
+    sleep::mayastor_sleep,
+    store::store_defs::StoreError,
+};
+use etcd_client::{Compare, CompareOp, Error as EtcdErr, TxnOp, TxnOpResponse};
```

Comment:
this is breaking the `PersistentStore` API, which will complicate replacing it with a proxy (there's a ticket for that somewhere).
We should abstract this behavior inside the `PersistentStore`